src/include/cart/api.h

```diff
@@ -523,6 +523,7 @@ crt_ep_abort(crt_endpoint_t *ep);
 #define CRT_VAR   0
 #define CRT_PTR   1
 #define CRT_ARRAY 2
+#define CRT_RAW   3
```

Comment:
Since this modifies public api by adding new type, this will require minor version bump in Sconscript, cart.spec and debian/changelog files

Reply:
I have changed **minor** version in this patch. It can be viewed in the second commit which GitHub did for me.